ui/apps/platform/package.json

```diff
@@ -183,6 +181,80 @@
         "jest-resolve": "^29.7.0",
         "jest-watch-typeahead": "^2.2.2"
     },
+    "overrides": {
+        "@jest/types": "^29.6.3",
+        "@typescript-eslint/eslint-plugin": "^6.21.0",
+        "@typescript-eslint/parser": "^6.21.0",
+        "autoprefixer": "10.4.5",
+        "babel-jest": "^29.7.0",
+        "connected-react-router": {
+          "react": "^18.0.0"
+        },
+        "eslint-plugin-jest": "^27.6.0",
+        "eslint-plugin-jest-dom": {
+          "@testing-library/dom": "^10.1.0"
+        },
+        "eslint-plugin-testing-library": "^6.2.0",
+        "jest": "^29.7.0",
+        "jest-resolve": "^29.7.0",
+        "jest-watch-typeahead": "^2.2.2",
+        "monaco-yaml": {
+          "monaco-editor": "0.34.1"
+        },
+        "react-collapsible": {
+          "react": "^18.0.0",
+          "react-dom": "^18.0.0"
+        },
+        "react-copy-to-clipboard": {
+          "react": "^18.0.0"
+        },
```

Comment:
We can specifiy `"react": "^18.0.0"` globally at the top level here, but I opted to override it explicitly for each package so that the out of date deps are more visible.